app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java

```diff
@@ -66,12 +67,14 @@ protected void onCreate(Bundle savedInstanceState) {
     View contentView = LayoutInflater.from(this).inflate(R.layout.search, null);
     setContentView(contentView);
 
+    if (savedInstanceState != null) {
+        searchText = savedInstanceState.getString("text");
```

Comment:
2 space indentation

Reply:
You meant before "if" statement? I cross checked it, but the indentation seems to be correct, can you please explain it?

Reply:
Before `searchText` you're using 4 space tab, you should refactor it to 2 space tab.

Reply:
Okay @abdulwd @brijeshshah13 got it 👍